commands/build/lib/build.js

```diff
@@ -36,11 +53,23 @@ const config = ({
   core,
 } = {}) => {
   const CWD = argv.cwd || cwd;
+  const carbon = doINeedACarbonCopy(argv);
   let dir = CWD;
   let pkg = require(path.resolve(CWD, 'package.json')); // eslint-disable-line
   const corePkg = core ? require(path.resolve(core, 'package.json')) : null; // eslint-disable-line
+  pkg = carbon ? require(path.resolve('./react-native', 'package.json')) : pkg; // eslint-disable-line
```

Comment:
instead of having this path hardcoded to `./react-native/package.json`, could we let the user specify the path in a similar way as we do for the `core` package?

Reply:
Done!